spring-data-cassandra/src/test/java/org/springframework/data/cassandra/domain/AllPossibleTypes.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.cassandra.domain;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.net.InetAddress;
+import java.nio.ByteBuffer;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import org.springframework.data.cassandra.mapping.CassandraType;
+import org.springframework.data.cassandra.mapping.PrimaryKey;
+import org.springframework.data.cassandra.mapping.Table;
+import org.springframework.data.cassandra.test.integration.mapping.types.CassandraTypeMappingIntegrationTest.Condition;
+
+import com.datastax.driver.core.DataType.Name;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * @author Mark Paluch
+ */
+@Table
+@Data
+@NoArgsConstructor
+@RequiredArgsConstructor
+public class AllPossibleTypes {
```

Comment:
The `timestamp` column is tested with `RepositoryReturnTypesIntegrationTests.shouldReturnTimestamp`. I align test names in `MappingCassandraConverterUnitTests` to reflect `date` (`com.datastax.driver.core.LocalDate`) and `timestamp` (`java.util.Date`) tests.